tests/test_unique.py

```diff
@@ -0,0 +1,261 @@
+from simple_ddl_parser import DDLParser
+
+
+def test_unique_statement_in_columns():
```

Comment:
2/3 of these are copy-paste from test_simple_ddl_parser.py.  The last test here is the new one for `UNIQUE KEY` statements.